### PR TITLE
Actions to build on 3 os

### PR DESCRIPTION
**Pull request by jburel**
Build and run the tests on 3 os using GH actions
cc @sbesson

### CODE REVIEW

**Review by manics — APPROVED** [review]
Looks OK to me as a first step.

Does this build the installers too, or just the jars and launch script?

**Comment by jburel** [comment]
Only the jars and the launch script.
For the dmg,`` javapackager`` that we rely on is not installed. It has also been removed from Java 11 (oracle).
The community is working on a new packager: ``jpackage``, so we might have to review our packaging strategy

**Review by joshmoore — APPROVED** [review]
Generally, :+1: I assume this doesn't change anything in our release process, right?

Also, the `name:` of the workflow can be used to have a badge URL if that's something you want to enable in later PRs.

**Comment by jburel** [comment]
Travis is still used for the release, This is a first step.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+name: Build with Gradle and run the tests
```
Comment (sbesson) [inline .github/workflows/main.yml:2]: Given its usage via the GitHub API e.g. badges, I have been leaning towards giving workflows concise single-word names and using the job name for additional verbosity - see e.g. https://github.com/ome/.github/blob/c8e7d8e8373f02cd956cd95e176f9aa2c0d41b74/workflow-templates/omero_plugin.yml#L10-L18

Comment (jburel) [reply .github/workflows/main.yml:2]: I will name it ``Build``

**Comment by joshmoore** [comment]
:+1:

**Review by sbesson — APPROVED** [review]

**Pull request merged**